### PR TITLE
0.5.0 release notes

### PR DESCRIPTION
**Pull request by yaron2**
Release notes for 0.5.0

### CODE REVIEW

File: docs/release_notes/v0.5.0.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+
+You can now follow [these](https://github.com/dapr/docs/blob/master/getting-started/environment-setup.md#using-helm-advanced) instructions on how to install Dapr using __Helm 3__.
+
+Alternatively, if you want to install Dapr in a dev/test setup, run:
+
+```
+dapr init --kubernetes
+```
+
+## Breaking Changes
```
Comment (LMWF) [inline docs/release_notes/v0.5.0.md]: There is another one here, but most users won't see it.  If a user invokes daprd manually (some users have mentioned doing this), --app-id changed to --dapr-id.  This was mentioned above but not as a breaking change.

Comment (yaron2) [reply docs/release_notes/v0.5.0.md]: We still don't have docs on `daprd`.. so we can't really address it here. but I'll add it to breaking changes anyway.

Comment (yaron2) [reply docs/release_notes/v0.5.0.md]: Added!

**Comment by LMWF** [comment]
LGTM

File: docs/release_notes/v0.5.0.md [hunk 2]
```diff
@@ -0,0 +1,241 @@
+
+  
+# Dapr 0.5.0
+ 
+We're happy to announce the release of Dapr 0.5.0! 
+
+This release introduces a *new* Secrets API to get application secrets, comprehensive metrics for the Dapr runtime as well as Dapr's system components, the ability to scope components to a namespace and/or a list of authorized apps, improved logging with JSON output, tracing support for Pub/Sub, new components, retries for bindings and general bugs fixes and stability improvements.
```
Comment (pruthvidhodda) [inline docs/release_notes/v0.5.0.md]: nit: bugs fixes -> bug fixes

File: docs/release_notes/v0.5.0.md [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+
+### Java SDK
+
+**Fixed** Hot monos in gRPC client
+
+**Added** Secrets API support
+
+**Fixed** Swallowing actor exceptions
+
+**Aded** Strongly typed actors
```
Comment (artursouza) [inline docs/release_notes/v0.5.0.md]: Added

Comment (yaron2) [reply docs/release_notes/v0.5.0.md]: Fixed

File: docs/release_notes/v0.5.0.md (hunk 2 above)
Comment (yaron2) [reply docs/release_notes/v0.5.0.md]: Fixed

**Pull request merged**